src/components/main/StaffList.js

```diff
@@ -4,12 +4,56 @@ import './StaffList.css';
 
 class StaffList extends React.Component {
    
-  
```

Comment:
Can we change some of the variable names here? What is sf? Is that Staff type?

Reply:
I agree. I also don't understand what sf is used for since it's only passed in to the function but it appears nothing is done with it. Renaming it to something clearer of what it does would help me understand.